src/AppBundle/Service/TestService.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: arnoldas
+ * Date: 16.11.20
+ * Time: 10.27
+ */
+
+namespace AppBundle\Service;
+use Doctrine\ORM\EntityManager;
+use Symfony\Component\Config\Definition\Exception\Exception;
+
+class TestService
+{
+    /**
+     * @var EntityManager
+     */
+    private $em;
+
+    public function __construct($em)
+    {
+        $this->em = $em;
+    }
+
+    public function getTestById($id)
+    {
+        $repository = $this->em->getRepository('AppBundle:Test');
+        $posts = $repository->find($id);
+
+        return $posts;
+    }
+    public function getTests($name){
+
+        $repository = $this->em->getRepository('AppBundle:Test');
+        $query = $repository->createQueryBuilder('p')
+            ->where('p.name LIKE :name')
+            ->orWhere('p.description LIKE :name')
+            ->setParameter('name', '%'.$name.'%')
+            ->getQuery();
+        $tests_result = $query->getResult();
+
+
+        return $tests_result;
+    }
+    public function get5RecentTests(){
```

Comment:
I would suggest doing it as getRecentTests($count = 5) that way, method would be more reusable, naming would be better, and default behavior calling getRecentTests() would still return 5 due to default value